src/Npgsql/TypeMapping/TypeMapperBase.cs

```diff
@@ -96,6 +97,30 @@ public INpgsqlTypeMapper MapComposite<T>(string? pgName = null, INpgsqlNameTrans
             }.Build());
         }
 
+        public INpgsqlTypeMapper MapComposite(Type compType, string pgName, INpgsqlNameTranslator? nameTranslator = null)
+        {
+            if (string.IsNullOrWhiteSpace(pgName))
+                throw new ArgumentException("pgName can't be empty", nameof(pgName));
```

Comment:
What is a reason to change method behavior? From the user point of view a generic method should work like a non-generic one.

Reply:
corrected